mmdeploy/codebase/mmdet/deploy/object_detection.py

```diff
@@ -25,6 +25,10 @@ class MMDetection(MMCodebase):
     @classmethod
     def register_all_modules(cls):
         from mmdet.utils.setup_env import register_all_modules
+
+        import mmdeploy.codebase.mmdet.models  # noqa: F401
```

Comment:
Can we wrap the import into a method? so we can register it in mmocr instead of copy these statement.